### PR TITLE
ticket 11

### PR DESCRIPTION
**Pull request by Gazdean**

### CODE REVIEW

File: __tests__/api.test.js [hunk 1]
```diff
@@ -91,8 +91,25 @@ describe('GET /api/articles', () => {
...
+        .get('/api/articles?topic=mitch')
+        .expect(200)
+        .then(({ body }) => {
+            const articles = body.articles
+            expect(articles.length).toBe(12)
+        })
+    })
+    test('when queried with an invalid topic it returns an empty array and status code 200', () => {
+        return request(app)
+        .get('/api/articles?topic=invalidTopic')
```
Comment (pippaamy) [inline __tests__/api.test.js]: non existent topic would be a 404

File: __tests__/api.test.js [hunk 2]
```diff
@@ -91,8 +91,25 @@ describe('GET /api/articles', () => {
             expect(articles).toBeSortedBy('created_at', {descending: true})
         })
     })
+    test('when queried with a topic value it responds status code 200 and with all article of that topic', () => {
+        return request(app)
+        .get('/api/articles?topic=mitch')
+        .expect(200)
+        .then(({ body }) => {
+            const articles = body.articles
```
Comment (pippaamy) [inline __tests__/api.test.js:98]: what if the topic=paper which id valid but there are no articles

File: db/controllers/articles-controller.js [hunk 3]
```diff
@@ -1,8 +1,14 @@
 const {fetchArticles, fetchArticlesById, fetchAllArticleComments, createArticleComment, updateArticleById, fetchAllUsers, removeCommentById} = require('../models/articles-model')
 exports.getArticles = (req, res, next) => {
-    fetchArticles().then((articles) => {
+    console.log(req.query)
```
Comment (pippaamy) [inline db/controllers/articles-controller.js]: remember to take out console logs

**Review by pippaamy — COMMENTED** [review]
good job, some changes to make! no need for another pr :)

File: db/controllers/articles-controller.js (hunk 3 above)
Comment (Gazdean) [reply db/controllers/articles-controller.js]: done

File: __tests__/api.test.js (hunk 2 above)
Comment (Gazdean) [reply __tests__/api.test.js:98]: i think that is what i meant by invalid topic, does this test cover that? 
Ive returned an empty array
Ive altered the wording of the test.

**Comment by Gazdean** [comment]
merged
branch deleted

**Pull request merged**